packages/sources/svelte/src/components/indicators/VtmnRating/VtmnRating.svelte

```diff
@@ -72,7 +72,11 @@
   $: starsCnt = compact && readonly ? 1 : 5;
 </script>
 
-<div class={componentClass} aria-disabled={disabled} {...$$restProps}>
+<div
+  class={componentClass}
+  aria-disabled={disabled}
+  {...readonly && $$restProps}
```

Comment:
same issue here + should you add parentheses here?

Reply:
yes indeed